modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -297,7 +297,7 @@ public List<ServiceRegistration<?>> deploy(
 			_bundleContext.registerService(
 				PortletResourcePermission.class, portletResourcePermission,
 				HashMapDictionaryBuilder.<String, Object>put(
-					"com.liferay.object", "true"
+					"com.liferay.object.definition", "true"
```

Comment:
Could you please check if we can use here the `ObjectConstants.RESOURCE_NAME_OBJECT_DEFINITION`?